### DISCUSSION TITLE
What is the expected way to add bindings to the FreqTableSheet in the develop?

### DISCUSSION THREAD

**Discussion by frosencrantz** [saulpw/visidata#1731] [Q&A]
It looks like the FreqTableSheet is no longer a global, this call doesn’t work anymore, `FreqTableSheet.addCommand` like in AJ's code
https://github.com/ajkerrigan/visidata-plugins/blob/main/plugins/parent_navigation.py#L107-L109

**Comment by saulpw** [comment]
Sorry, I've added it back.
